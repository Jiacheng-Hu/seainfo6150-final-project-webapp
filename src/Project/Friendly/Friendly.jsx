import React from "react";

const Friendly = () => {

    return (
        <div className="container">
            <h1  className="text-center">Cooperative Partner</h1>
            <div className="row text-center">
                <div className="col-xs-12 col-sm-6 col-md-4 col-lg-3"><a href="http://www.github.com">Github</a></div>

                <div className="col-xs-12 col-sm-6 col-md-4 col-lg-3"><a href="http://www.google.com">Google</a></div>
                
                <div className="col-xs-12 col-sm-6 col-md-4 col-lg-3"><a href="http://www.stackoverflow.com">Stack Overflow</a></div>
                
                <div className="col-xs-12 col-sm-6 col-md-4 col-lg-3"><a href="http://www.w3schools.com">W3Schools</a></div>
                
                <div className="col-xs-12 col-sm-6 col-md-4 col-lg-3"><a href="http://www.stackexchange.com">Stack Exchange</a></div>
                
                <div className="col-xs-12 col-sm-6 col-md-4 col-lg-3"><a href="http://www.reddit.com">Reddit</a></div>

                <div className="col-xs-12 col-sm-6 col-md-4 col-lg-3"><a href="http://www.udemy.com">Udemy</a></div>
                
                <div className="col-xs-12 col-sm-6 col-md-4 col-lg-3"><a href="http://www.youtube.com">YouTube</a></div>
                
                <div className="col-xs-12 col-sm-6 col-md-4 col-lg-3"><a href="www.facebook.com">Facebook</a></div>
            </div>
        </div>
    )
}

export default Friendly;