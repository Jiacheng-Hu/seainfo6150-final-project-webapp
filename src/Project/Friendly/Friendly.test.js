import React from "react";
import Friendly from "./Friendly";


describe("Friendly tests", () => {
    it("renders correctly", () => {
        const {container} = render(<Friendly/>);
        expect(container).toMatchSnapshot();
    });
});