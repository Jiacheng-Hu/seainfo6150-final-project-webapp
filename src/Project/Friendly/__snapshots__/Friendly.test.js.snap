// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Friendly tests renders correctly 1`] = `
<div>
  <div
    class="container"
  >
    <h1
      class="text-center"
    >
      Cooperative Partner
    </h1>
    <div
      class="row text-center"
    >
      <div
        class="col-xs-12 col-sm-6 col-md-4 col-lg-3"
      >
        <a
          href="http://www.github.com"
        >
          Github
        </a>
      </div>
      <div
        class="col-xs-12 col-sm-6 col-md-4 col-lg-3"
      >
        <a
          href="http://www.google.com"
        >
          Google
        </a>
      </div>
      <div
        class="col-xs-12 col-sm-6 col-md-4 col-lg-3"
      >
        <a
          href="http://www.stackoverflow.com"
        >
          Stack Overflow
        </a>
      </div>
      <div
        class="col-xs-12 col-sm-6 col-md-4 col-lg-3"
      >
        <a
          href="http://www.w3schools.com"
        >
          W3Schools
        </a>
      </div>
      <div
        class="col-xs-12 col-sm-6 col-md-4 col-lg-3"
      >
        <a
          href="http://www.stackexchange.com"
        >
          Stack Exchange
        </a>
      </div>
      <div
        class="col-xs-12 col-sm-6 col-md-4 col-lg-3"
      >
        <a
          href="http://www.reddit.com"
        >
          Reddit
        </a>
      </div>
      <div
        class="col-xs-12 col-sm-6 col-md-4 col-lg-3"
      >
        <a
          href="http://www.udemy.com"
        >
          Udemy
        </a>
      </div>
      <div
        class="col-xs-12 col-sm-6 col-md-4 col-lg-3"
      >
        <a
          href="http://www.youtube.com"
        >
          YouTube
        </a>
      </div>
      <div
        class="col-xs-12 col-sm-6 col-md-4 col-lg-3"
      >
        <a
          href="www.facebook.com"
        >
          Facebook
        </a>
      </div>
    </div>
  </div>
</div>
`;
